src/Data/Fin/Subset/Properties.agda

```diff
@@ -152,13 +152,12 @@ x∈⁅y⁆⇔x≡y {_} {x} {y} = equivalence
 infix 4 _⊆?_
 _⊆?_ : ∀ {n} → B.Decidable (_⊆_ {n = n})
 []          ⊆? []          = yes id
-outside ∷ p ⊆? y ∷ q with p ⊆? q
-... | yes p⊆q = yes λ { (there v∈p) → there (p⊆q v∈p)}
-... | no  p⊈q = no (p⊈q ∘ drop-∷-⊆)
+outside ∷ p ⊆? y ∷ q =
+  map′ (λ { p⊆q (there v∈p) → there (p⊆q v∈p) }) drop-∷-⊆ (p ⊆? q)
```

Comment:
Could we split the first direction into it's own lemma for these two?

Reply:
How about these lemmas being equivalences, and then using `map` rather than `map′`?

```agda
outside∷-⊆-⇔ : ∀ {n y} {p q : Subset n} → p ⊆ q ⇔ outside ∷ p ⊆ y ∷ q
outside∷-⊆-⇔ = equivalence (λ { p⊆q (there ∈p) → there (p⊆q ∈p) }) drop-∷-⊆
```

Reply:
Hmm except that then you have to get `Setoid`s involved unnecessarily which feels like overkill. Not opposed to adding a lemma that they are an `Equivalence` of course (just make sure you use the new `Function` hierarchy rather than the old one!).

Reply:
Aah, okay. I hadn't seen this comment when I added the commits earlier, so I'll look over them again.

Reply:
Okay, no worries!